### ISSUE TITLE
Can't buy a share with the Ledger Nano S

### ISSUE THREAD

**Issue by IvanBoutin** [AugurProject/augur-ui#2103]
Hi,

I tried to buy a share of a event on an augur market with my Ledger Nano S. It's working to step of signing the transaction with my ledger, but after the signing, nothing happens. No transaction on EtherScan, no alert in the grey alert window of the augur UI. I was able to approve augur and buy participations tokens with my ledger. This seems a problem with the buying/selling interface only.

My Setup : 
Augur node 1.0.7 on Windows 10
Sync my local parity node
Ledger Nano S for the wallet
Browser Chrome 68.0.3440.75

**Comment by bthaile** [comment]
Did your trade transaction ever show up or go through?

Its possible the default gas could have been too low and never got mined.

**Comment by IvanBoutin** [comment]
No, i tried multiple times last week, and every transactions I made never showed up in etherscan. If the gas is too, i still supposed to see transaction in etherscan no?

**Comment by bthaile** [comment]
Do you have an etherscan of your account. Maybe we can see something.

**Comment by IvanBoutin** [comment]
Sure, there it is :

https://etherscan.io/address/0x3f071A22e6e92D2AfC478D8196d1cd2Aa29B4D2D

**Comment by IvanBoutin** [comment]
Ok, I found the problem.  I was able to make a trade with less transaction value. I didn't have enough ether in my account to paid for the transaction and the estimated gas price. I see it clearly in the webconsole. It will be interesting to have an error in the UI instead only in webconsole. 

Error in webconsole : {code: -32603, message: "Error: insufficient funds for gas * price + value↵…ogaeaoehlefnkodbefgpgknn/background.js:1:1070338)"}
code
:
-32603
message
:
"Error: insufficient funds for gas * price + value↵    at chrome-
